packages/office-ui-fabric-react/src/components/Button/examples/Button.CustomSplit.Example.tsx

```diff
@@ -50,6 +50,7 @@ export const ButtonSplitCustomExample: React.FunctionComponent<IButtonExamplePro
         menuProps={menuProps}
         // This text is hidden by the styles
         text="New item"
+        title="New item"
```

Comment:
I'm confused, why does this need a title when it has text? Seems like a bug in IconButton if it's not either properly labeling its icon, or marking it as hidden.

Reply:
Never mind, I see what you're getting at now, but this approach still seems kind of weird and like it shouldn't be necessary. On the surface it SEEMS like all you should need is a label for the main button (`text`), and a label for the split button (`splitButtonAriaLabel`). And if those aren't getting read out properly, it's an issue with how the props are being applied inside the button.

Reply:
There shouldn't even be a text here actually. Since the IconButton EXPLICITLY turn off rendering of text. I think maybe the right fix here would be to replace "text" with title not in addition.

Reply:
Text shouldn't end up being in the title for a button - that's redundant for all other buttons. So, we should just add a title prop in example.

Reply:
I'm still sort of confused--does the `title` get applied to the whole button, or only the split part (down arrow)? It seems to me like the main button and the split part should have different titles/labels.

Reply:
@jspurlin might be interested in this convo

Reply:
I'm very close to just deleting this example - because using an IconButton to simulate a SplitButton seems like a really easy way to create a inaccessible SplitButton. Is this what we want to show?

Reply:
The whole splitButton should have at most one tooltip (e.g. title) because it is one (compound) component. It doesn't make sense for multiple portions of the same button to have different labels. Note, ComboBox is another instance of a compound component that contains a button, if it has a tooltip it's set on the whole component (not separate tooltips for the input portion vs the button portion)... it's perceived as one component.

---
In reply to: [325955819](https://github.com/OfficeDev/office-ui-fabric-react/pull/10498#discussion_r325955819) [](ancestors = 325955819)

Reply:
`title` ends up on the correct button in any case actually. So, in my eyes, it is the right fix. It might not be obvious what's going on here with the props.

Reply:
This could potentially put the text on the title if it is an IconButton (and there is not a title supplied) since the text wouldn't be used otherwise, correct?

Reply:
The fix would look something like "if onTextRender is null renderer, and there is a text prop but no title prop, do this" - does that logic sound okay, @jspurlin?

Reply:
sounds good to me